extensions/wikia/Search/classes/Services/ESFandomSearchService.php

```diff
@@ -34,7 +36,7 @@ protected function select( $query ) {
 		if ( $response !== false ) {
 			$decodedResponse = json_decode( $response, true );
 			if ( json_last_error() !== JSON_ERROR_NONE ) {
-				\WikiaLogger::instance()->error(
+				WikiaLogger::instance()->error(
 					" Fandom Stories Search: error decoding response",
 					[
 						'query' => $query,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "query" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)